pom.xml

```diff
@@ -25,6 +25,7 @@
 	</properties>
 	<modules>
 		<module>spring-cloud-stream</module>
+		<module>spring-cloud-stream-rxjava</module>
```

Comment:
Possibly, but even projects like SI host stable extensions as part of the main project. Since Rx support is something we definitely want as part of core functionality, I think it's ok to have it here.